### PR TITLE
Np multiome wrapper wdl

### PR DESCRIPTION
**Pull request by nikellepetrillo**
### Description

_Give your PR a **concise** yet **descriptive** title._
_Please explain the changes you made here._
_Explain the **motivation** for making this change. What existing problem does the pull request solve?_
_Mention any issues fixed, addressed, or otherwise related to this pull request, including issue numbers or hard links for issues in other repos._
_You can delete these instructions once you have written your PR description._

----

### Checklist 
If you can answer "yes" to the following items, please add a checkmark next to the appropriate checklist item(s) **and** notify our WARP documentation team by tagging either @ekiernan or @kayleemathews in a comment on this PR.

- [ ] Did you add inputs, outputs, or tasks to a workflow?
- [ ] Did you modify, delete or move: file paths, file names, input names, output names, or task names?
- [ ] If you made a changelog update, did you update the pipeline version number?

### CODE REVIEW

**Comment by nikellepetrillo** [comment]
multiome wrapper run --> https://job-manager.dsde-prod.broadinstitute.org/jobs/a3b2857b-2fe4-4303-af85-e8296364ef78

File: pipelines/skylab/optimus/Optimus.changelog.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# 5.7.5
+2023-04-19 (Date of Last Commit)
+
+* Updated warp-tools docker which included a fix for a small bug in create_snrna_optimus.py that was causing the script not to run.
```
Comment (ekiernan) [inline pipelines/skylab/optimus/Optimus.changelog.md:4]:
```suggestion
* Updated warp-tools docker which included a fix for a small bug in create_snrna_optimus.py that was causing the script not to run

File: pipelines/skylab/slideseq/SlideSeq.changelog.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+# 1.0.4
+2023-04-19
+* Updated warp-tools docker which included a fix for a small bug in create_snrna_optimus.py that was causing the script not to run.
```
Comment (ekiernan) [inline pipelines/skylab/slideseq/SlideSeq.changelog.md:3]:
```suggestion
* Updated warp-tools docker which included a fix for a small bug in create_snrna_optimus.py that was causing the script not to run

File: pipelines/skylab/smartseq2_multisample/MultiSampleSmartSeq2.changelog.md [hunk 3]
```diff
@@ -1,3 +1,8 @@
+# 2.2.21
+2023-04-19 (Date of Last Commit)
+
+* Updated warp-tools docker which included a fix for a small bug in create_snrna_optimus.py that was causing the script not to run.
```
Comment (ekiernan) [inline pipelines/skylab/smartseq2_multisample/MultiSampleSmartSeq2.changelog.md:4]:
```suggestion
* Updated warp-tools docker which included a fix for a small bug in create_snrna_optimus.py that was causing the script not to run

File: pipelines/skylab/smartseq2_single_nucleus_multisample/MultiSampleSmartSeq2SingleNucleus.changelog.md [hunk 4]
```diff
@@ -1,3 +1,8 @@
+# 1.2.21
+2023-04-19 (Date of Last Commit)
+
+* Updated warp-tools docker which included a fix for a small bug in create_snrna_optimus.py that was causing the script not to run.
```
Comment (ekiernan) [inline pipelines/skylab/smartseq2_single_nucleus_multisample/MultiSampleSmartSeq2SingleNucleus.changelog.md:4]:
```suggestion
* Updated warp-tools docker which included a fix for a small bug in create_snrna_optimus.py that was causing the script not to run

File: pipelines/skylab/smartseq2_single_sample/SmartSeq2SingleSample.changelog.md [hunk 5]
```diff
@@ -1,3 +1,8 @@
+# 5.1.20
+2023-04-19 (Date of Last Commit)
+
+* Updated warp-tools docker which included a fix for a small bug in create_snrna_optimus.py that was causing the script not to run.
```
Comment (ekiernan) [inline pipelines/skylab/smartseq2_single_sample/SmartSeq2SingleSample.changelog.md:4]:
```suggestion
* Updated warp-tools docker which included a fix for a small bug in create_snrna_optimus.py that was causing the script not to run

**Review by ekiernan — APPROVED** [review]
Made minor fixes to the changelog entries. LGTM! Thank you.

**Pull request merged**